eth/protocols/eth/handlers.go

```diff
@@ -260,12 +277,12 @@ func handleNewBlock(backend Backend, msg Decoder, peer *Peer) error {
 		return nil // TODO(karalabe): return error eventually, but wait a few releases
 	}
 	// Dom nodes need to validate the subordinate manifest against the subordinate's manifesthash
-	if nodeCtx < common.ZONE_CTX {
-		if hash := types.DeriveSha(ann.Block.SubManifest(), trie.NewStackTrie(nil)); hash != ann.Block.ManifestHash(nodeCtx+1) {
-			log.Warn("Propagated block has invalid body", "have", hash, "exp", ann.Block.ManifestHash())
-			return nil // TODO(karalabe): return error eventually, but wait a few releases
-		}
-	}
+	// if nodeCtx < common.ZONE_CTX {
```

Comment:
Delete this comment block.

Reply:
deleted it and made an issue to come back. https://github.com/dominant-strategies/go-quai/issues/545